### PR TITLE
Add BreadBuddy to projects.json

### PR DESCRIPTION
**Pull request by maxhumber**
### Pull Request Description

[BreadBuddy](https://github.com/maxhumber/BreadBuddy) is an open-source reference app for using MVVM with SwiftUI.

### Acceptance Criteria

To be accepted into the Swift source compatibility test suite, a project must:

- [x] be an **Xcode** or *swift package manager* project
- [x] support building on either Linux or **macOS**
- [x] target Linux, macOS, or **iOS**/tvOS/watchOS device
- [x] be contained in a publicly accessible git repository
- [x] maintain a project branch that builds against Swift 4.0 and passes any unit tests
- [x] have maintainers who will commit to resolve issues in a timely manner
- [x] be compatible with the latest GM/Beta versions of *Xcode* and *swiftpm*
- [x] add value not already included in the suite
- [x] be licensed with one of the following permissive licenses:
	* **BSD**
	* MIT
	* Apache License, version 2.0
	* Eclipse Public License
	* Mozilla Public License (MPL) 1.1
	* MPL 2.0
	* CDDL
- [x] pass `./project_precommit_check` script run

Ensure project meets all listed requirements before submitting a pull request.

### CODE REVIEW

File: projects.json [hunk 1]
```diff
@@ -308,6 +308,30 @@
...
     ]
   },
+  {
+    "repository": "Git",
+    "url": "https://github.com/maxhumber/BreadBuddy",
+    "path": "BreadBuddy",
+    "branch": "master",
+    "maintainer": "max@bracket.ltd",
+    "compatibility": {
+      "5.6.1": {
```
Comment (shahmishal) [inline projects.json:318]: Can we update this to 5.0? we are trying to keep the compatibility to supported swift versions. 
* Swift 4.0
* Swift 4.2
* Swift 5.0

**Comment by shahmishal** [comment]
@swift-ci test

File: projects.json (hunk 1 above)
Comment (maxhumber) [reply projects.json:318]: Yeah, shouldn’t be a problem. Would you like me to adjust? Or leave it to you?

**Comment by maxhumber** [comment]
Just bumped it down to 5.0. Let me know if you need anything else!

**Comment by maxhumber** [comment]
> **Identified problems**: No identified problem
> No problems were identified. If you know why this problem occurred, please add a suitable Cause for it.

🤪

**Comment by justice-adams-apple** [comment]
Hi @maxhumber 👋 , looks like the build system had an issue finding the xcodeproj file specified in this PR
```
xcodebuild: error: '/Users/ec2-user/jenkins/workspace/swift-PR-source-compat-suite-test-macOS/swift-source-compat-suite/project_cache/BreadBuddy/BreadBuddy/BreadBuddy.xcodeproj' does not exist.
```

File: projects.json [hunk 2]
```diff
@@ -308,6 +308,31 @@
...
+        "commit": "28558470aa2ef75a96156ab00aa6d42d05eac884"
+      }
+    ],
+    "platforms": [
+      "Darwin"
+    ],
+    "actions": [
+      {
+        "action": "BuildXcodeProjectTarget",
+        "project": "BreadBuddy/BreadBuddy.xcodeproj",
```
Comment (shahmishal) [inline projects.json:329]:
```suggestion
        "project": "BreadBuddy.xcodeproj",
```

Comment (shahmishal) [reply projects.json:329]: This should fix the build issue.

**Comment by maxhumber** [comment]
Changes committed!

**Comment by justice-adams-apple** [comment]
@swift-ci test

**Pull request closed without merging**